charts/datasaur/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.1.0
```

Comment:
is it okay if we bump the minor version @daniel-jeremia ?

Reply:
usually we use bump patch version for updates like this, it wouldn't change anything, just a convention @tonoman3g @robbydrmwn

Reply:
ohh.. should I change it to 2.1.1 then? @daniel-jeremia

Reply:
ganti ajaa @robbydrmwn

Reply:
updated ya @tonoman3g @daniel-jeremia